lib/validate/clusterconfig.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validate
+
+import (
+	"github.com/gravitational/gravity/lib/storage/clusterconfig"
+
+	"github.com/gravitational/trace"
+)
+
+// ClusterConfiguration validates that the update can update existing
```

Comment:
Incomplete sentence?

Reply:
Even with `update` and `existing` being the variable names?